src/nft/ZoraCreator1155Impl.sol

```diff
@@ -59,7 +59,7 @@ contract ZoraCreator1155Impl is
     /// @notice Factory contract
     IFactoryManagedUpgradeGate internal immutable factory;
 
-    constructor(uint256 _mintFeeAmount, address _mintFeeRecipient, address _factory) MintFeeManager(_mintFeeAmount, _mintFeeRecipient) initializer {
+    constructor(address _rewardsManager, address _zoraRewardRecipient, address _factory) CreatorRewards(_rewardsManager, _zoraRewardRecipient) initializer {
```

Comment:
we should consider allowing this to be address(0) - and then having it default if that's set to a defaul IRewardsManager, to make it easier to work with